cmake/CMakeLists.txt

```diff
@@ -159,6 +159,10 @@ else (MSVC)
   set(LIB_PREFIX)
 endif (MSVC)
 
+if (UNICODE)
+  add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
Why do we need to pass UNICODE to the compiler (msvc in this case)? If the user want to enable unicode build, can they just enable it inside visual studio (the configuration page)?